libs/pong/pong.js

```diff
@@ -95,6 +94,7 @@ export function createPongGetHandler(client, coder) {
 export function createPongClickHandler(coder) {
   return async (params) => {
     const click = coder.decodeAndVerify(params.get("code"));
+    // eslint-disable-next-line n/no-unsupported-features/node-builtins
```

Comment:
Nit: These ignores shouldn't be necessary, as we use Node.js 22, and fetch is [no longer experimental since Node 21.0.0](https://nodejs.org/docs/v22.18.0/api/globals.html#fetch).

Reply:
Eslint falls back to node 16 for me. Fixable, but not in this context.